tests/unit/src/main/java/org/mule/tck/AbstractTxThreadAssociationTestCase.java

```diff
@@ -1,324 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.tck;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-
-import org.mule.runtime.core.api.execution.ExecutionCallback;
-import org.mule.runtime.core.api.execution.ExecutionTemplate;
-import org.mule.runtime.core.api.transaction.TransactionConfig;
-import org.mule.runtime.core.api.transaction.TransactionManagerFactory;
-import org.mule.runtime.core.api.execution.TransactionalExecutionTemplate;
-import org.mule.tck.junit4.AbstractMuleContextTestCase;
-import org.mule.runtime.core.api.transaction.MuleTransactionConfig;
-import org.mule.runtime.core.privileged.transaction.XaTransaction;
-import org.mule.runtime.core.privileged.transaction.xa.XaTransactionFactory;
-
-import javax.transaction.Status;
-import javax.transaction.Transaction;
-import javax.transaction.TransactionManager;
-
-import org.junit.Test;
-
-/**
- * Validate certain expectations when working with JTA API. It is called to catch discrepancies in TM implementations and alert
- * early. Subclasses are supposed to plug in specific transaction managers for tests.
- */
-public abstract class AbstractTxThreadAssociationTestCase extends AbstractMuleContextTestCase {
```

Comment:
why remove this?

Reply:
It is an abstract class but is not extended. I've asked you before

Reply:
right!